### ISSUE TITLE
Warning on every .js file: `disallowMultipleLineBreaks`

### ISSUE THREAD

**Issue by razvan-tudosa** [AtomLinter/linter-jscs#120]
I get this warning on every javascript file at the beginning of the file: `disallowMultipleLineBreaks`. I should also mention that I have no line breaks at the beginning of my files.

As far as I know this rule says that I shouldn't have 2 or more line breaks in a row. [JSCS - Disallow Multiple Line Breaks](http://jscs.info/rule/disallowMultipleLineBreaks)

Does anybody have this issue also or has any clue where this problem could be from?

**Comment by simonappelt** [comment]
Which System do you use? Windows, Mac or Linux? Probably a problem with the different newline symbols.
Can you probably create a Gist or upload the file another way? Because so we can't recreate the error and can't find the bug.
Thanks

**Comment by razvan-tudosa** [comment]
I'm using Mac OSX Yosemite version 10.10.5.
In [this](https://gist.github.com/razvan-tudosa/8751390f0e0b20930835) gist I posted one of my files, just copy pasted it. I get the warning on the first line.

**Comment by simonappelt** [comment]
I've tried to recreate your error but I don't get it. Probably you messed up the invisibles? All presets worked for me.
Have you tried to create a new file and pasted the GIST in it? Did you still get it?

**Comment by casesandberg** [comment]
Are you still having this issue? Try updating `atom` and `linter-jscs` and let me know if its still an issue.

**Comment by razvan-tudosa** [comment]
Hello! Unfortunately I still have the problem. I have `atom` version 1.4.0 and `linter-jscs` version 3.3.0.
I put my `.jscsrc` file in [this](https://gist.github.com/razvan-tudosa/6aa196fba7b8282178fe) gist, if you wanna take a look.

**Comment by casesandberg** [comment]
Ah! I just used your `.jscsrc` and the error is actually `Unsupported rule: validateDJDoc`. If you remove that it works just fine. Im not sure if maybe that was a supported rule in the past, but it is what is causing your issue.

```
"validateJSDoc": {
        "checkParamNames": true,
        "requireParamTypes": true
    },
```

**Comment by razvan-tudosa** [comment]
@casesandberg This is wonderful! Thanks a lot!
